test/functional/wallet_avoidreuse.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the avoid_reuse feature."""
```

Comment:
It'd be nice if this tested the `setwalletflag` RPC too.

Reply:
It sort of does, actually. I assumed there were tests elsewhere, but apparently not. I will add complementary tests for setwalletflag.